.travis.yml

```diff
@@ -49,7 +49,7 @@ before_script:
   - sh -c "if [ '$HHVM' != '1' ]; then phpenv config-rm xdebug.ini; fi"
   - composer self-update
   - if [ -n "$GH_TOKEN" ]; then composer config github-oauth.github.com ${GH_TOKEN}; fi;
-  - composer install --prefer-dist --no-interaction
+  - composer install --prefer-source --no-interaction
 
```

Comment:
Because travis keeps failing every now and then due to github blocking the download API